src/applications/find-forms/actions/index.js

```diff
@@ -70,13 +104,12 @@ export const fetchFormsThunk = (query, options = {}) => async (
   try {
     // Attempt to make the API request to retreive forms.
     const resultsDetails = await fetchFormsApi(query, { mockRequest });
-
+    const resultsSorted = resultsDetails.results?.sort((a, b) =>
+      sortTheResults(getState().howToSort, a, b),
```

Comment:
`howToSort` is okay but I think `sortByPropertyName` could have been clearer. `howTo` makes me think of "How to" articles or something.